database/migrations/sqlite/1718182822233211_deployment_attempts.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS deployment_attempts
+(
+    eslVersion INTEGER, -- internal ID for ESL
+    created TIMESTAMP,
+    envName VARCHAR,
+    appName VARCHAR,
+    queuedVersion BIGINT NULL,
```

Comment:
I think the other table calls it `releaseVersion`, so we should call it `queuedReleaseVersion` here.